test/libsolidity/semanticTests/userDefinedValueType/erc20.sol

```diff
@@ -115,7 +115,7 @@ contract ERC20 {
 // ----
 // constructor()
 // ~ emit Transfer(address,address,uint256): #0x00, #0x1212121212121212121212121212120000000012, 0x14
-// gas irOptimized: 442239
+// gas irOptimized: 447831
```

Comment:
We should (separately) investigate the increase in gas cost here and check if it can be avoided. But for now I'll approve and merge this PR.